src/UI/examples/Image/licence_images.txt

```diff
@@ -0,0 +1,2 @@
+All images taken from https://stocksnap.io
+All photos on StockSnap fall under the Creative Commons CC0 license. That means you can copy, modify, distribute any photo on the site, even for commercial purposes, all without asking permission!
```

Comment:
I think we need to show this message in the UI of the kitchen sink. Is this implemented?